integration-tests/runtime-tests/test/tests/pablo/testHandlers/pabloTestHelper.ts

```diff
@@ -0,0 +1,452 @@
+          quote: api.createType('u128', Math.floor(Math.random() * 10000))
+        }),
+        fee: api.createType('Permill', Math.floor(Math.random() * 100000)),
+        ownerFee: api.createType('Permill', Math.floor(Math.random() * 100000))
+      }
+    });
+    tx.push(api.tx.pablo.create(pool));
+  }
+  await sendWithBatchAndWaitForSuccess(api, wallet, api.events.pablo.PoolCreated.is, tx, false);
+}
+
+export async function getUserTokens(api: ApiPromise, walletId: KeyringPair, assetId: number): Promise<u128> {
+  const {free, reserved, frozen} = await api.query.tokens.accounts(walletId.address, assetId);
+  return free;
+}
+
+export async function getPoolInfo(
+  api: ApiPromise,
+  poolType: string,
+  poolId: number
+): Promise<{ ownFee: number, lpTokenId: number }> {
+  const result = await api.query.pablo.pools(api.createType('u128', poolId));
+  const pool = result.unwrap();
+  const poolS = "as" + poolType;
+  const ownFee = pool[poolS].ownerFee.toNumber();
+  const lpTokenId = pool[poolS].lpToken.toNumber();
+  return {ownFee, lpTokenId};
+}
+
+export async function rpcPriceFor(
+  api: ApiPromise,
+  poolId: PalletPabloPoolId,
+  baseAssetId: CustomRpcCurrencyId,
+  quoteAssetId: CustomRpcCurrencyId
+) {
+  return await api.rpc.pablo.pricesFor(
+    poolId,
+    baseAssetId,
+    quoteAssetId,
+    api.createType('CustomRpcBalance', 10000 /* unit */));
+}
+
+export async function getPoolAddress(
+  api: ApiPromise,
+  poolId: number,
+  walletId: KeyringPair,
+  baseAmount: bigint,
+  quoteAmount: bigint
+): Promise<any> {
+  const pool = api.createType("u128", poolId)
+  const baseAmountParam = api.createType('u128', baseAmount);
+  const quoteAmountParam = api.createType('u128', quoteAmount);
+  const keepAliveParam = api.createType('bool', true);
+  const minMintAmountParam = api.createType('u128', 0);
+  const {data: [, AccountId,]} = await sendAndWaitForSuccess(
+    api,
+    walletId,
+    api.events.tokens.Endowed.is,
+    api.tx.pablo.addLiquidity(pool,
+      baseAmountParam,
+      quoteAmountParam,
+      minMintAmountParam,
+      keepAliveParam
+    )
+  );
+  return AccountId.toString();
+}
+
+export async function getPoolBalance(api: ApiPromise, poolAddress: string, assetId: number): Promise<u128> {
+  const {free, reserved, frozen} = await api.query.tokens.accounts(poolAddress, assetId);
+  return free;
+}
+
+export async function transferTokens(api: ApiPromise, sender: KeyringPair, receiver: KeyringPair, assetId: number, amount: bigint)
+  : Promise<string> {
+  const {data: [, accountId,]} = await sendAndWaitForSuccess(
+    api,
+    sender,
+    api.events.tokens.Endowed.is,
+    api.tx.assets.transfer(
+      api.createType("u128", assetId),
+      api.createType("MultiAddress", {
+        id: api.createType("AccountId", receiver.address.toString()),
+      }),
+      api.createType("u128", amount),
+      api.createType("bool", false)
+    )
+  );
+  return accountId.toString();
+}
+/***
```

Comment:
Should we split this file up into 3, one for each type of dex, to make it more easily readable?

Reply:
It was initially under 3 files, I thought this would be better for readability :D But I can split them again.

Reply:
Don't know what would make the most sense here tbh.
For me both works fine.